Comp2024/.DataLogTool/datalogtool.json

```diff
@@ -0,0 +1,8 @@
+{
```

Comment:
When you see a tool settings file (like datalogtool.json), just skip over them rather than commit them. We should probably add them to .gitignore in the future.